base/timing.jl

```diff
@@ -117,6 +117,32 @@ function jit_total_bytes()
     return ccall(:jl_jit_total_bytes, Csize_t, ())
 end
 
+thread_up_time() = ccall(:jl_thread_up_time, UInt64, ())
```

Comment:
Now we have per-thread scheduler time, i think it would be good to expose a function for *global* scheduler time (like `Base.scheduler_time_ns()`), to complement `Base.gc_time_ns()` / `jl_gc_total_hrtime()` for GC time and `Base.cumulative_compile_time_ns()` /`jl_cumulative_compile_time` for compile time

Reply:
Sounds good. Will add it.

Reply:
Marking as completed, since will be implementing:

> maybe rather than having a user_time metric that minuses these various totals from a global start_time, we need to instead just have function to return the individual values scheduler_time_ns() etc, and then a user can minus these from a cpu metric coming from the kernel?